CHANGELOG.md

```diff
@@ -2,384 +2,398 @@
 
 This changelog is a work in progress and may contain notes for versions which have not actually been released. Check the [Releases](https://github.com/0xProject/0x-mesh/releases) page to see full release notes and more information about the latest released versions.
 
+## v10.0.0
+
+### Breaking changes 🛠
+
+-   Reject orders that have taker addresses that are non-zero and are not whitelisted. [#896](https://github.com/0xProject/0x-mesh/pull/896)
+-   Removed support for Coordinator orders. [#895](https://github.com/0xProject/0x-mesh/pull/895)
+-   The JSON-RPC is no longer supported by Mesh nodes. Only a GraphQL API is supported. [#858](https://github.com/0xProject/0x-mesh/pull/858)
```

Comment:
This should be up top!

Reply:
Also sounds a bit Dostoyevsky. How about "JSON-RPC supported is removed in favour of the a GraphQL API"/